torch/_dynamo/variables/user_defined.py

```diff
@@ -329,14 +329,16 @@ def call_function(
             and hasattr(
                 self.value, "__exit__"
             )  # TODO(voz): These can invoke user code!
-            and check_constant_args(args, kwargs)
-            and self.value.__init__ == object.__init__
-            and len(kwargs) == 0  # TODO(ybliang): support kwargs
+            and self.value.__new__ == object.__new__
         ):
-            unwrapped_args = [x.as_python_constant() for x in args]
+            cm_obj = tx.output.side_effects.track_object_new(
```

Comment:
@yanboliang are you saying there is a way for us to get attribute mutation working on a non-UserDefinedObjectVariable? (That's what we're trying to accomplish in this PR)

Reply:
Yes, actually ```UserDefinedObjectVariable``` is the most ultimate fallback variable, which has the least customized feature. So if sth is supported by UDO, it would not hard to make other variable tracker support it.

Reply:
Sounds good!